invenio_rdm_migrator/streams/actions/users.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Migrator is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""User actions module."""
+
+from dataclasses import dataclass
+from typing import Optional
+
+from ...actions import LoadAction, LoadData
+from ...load.postgresql.transactions.operations import Operation, OperationType
+from ...transform import EncryptMixin
+from ..models.users import LoginInformation, SessionActivity, User
+
+# GLOBAL FIXME: partial updates would allow to remove the encrypt mixin from many of the
```

Comment:
important

Reply:
We could use the kafka transformation https://debezium.io/documentation/reference/stable/transformations/event-changes.html or configure payloads to contain the `before` information and do a dict diff (which is what the kafka transformation does) but the responsability would be on us.

Reply:
Nice find! Yes that could work, though indeed we need to configure in PostgreSQL table with `REPLICA IDENTITY FULL` (https://www.postgresql.org/docs/current/logical-replication-publication.html)